plugins/lvmprovisioner/provisioner.go

```diff
@@ -0,0 +1,133 @@
+package lvmprovisioner
+
+import (
+	"errors"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/gluster/glusterd2/pkg/fsutils"
+	"github.com/gluster/glusterd2/pkg/lvmutils"
+)
+
+var mountOpts = "rw,inode64,noatime,nouuid"
```

Comment:
We also need 'discard' as a mount option? So that actual backend gets free'd up ?

Reply:
On LVM it returns space to the TP, and on loop, it'll return space to the outer file system... both are desirable.